packages/store-sync/benchmarks/query.bench.ts

```diff
@@ -0,0 +1,36 @@
+import { bench, describe } from "vitest";
+import { Has, runQuery } from "@latticexyz/recs";
+import { createRecsStorage, createSqliteStorage, createZustandStorage, tables } from "../test/utils";
+import { logsToBlocks } from "../test/logsToBlocks";
+import worldRpcLogs from "../../../test-data/world-logs-query.json";
+import { buildTable, getTables } from "../src/sqlite";
+import { eq } from "drizzle-orm";
+
+const blocks = logsToBlocks(worldRpcLogs);
```

Comment:
can we use the `generateLogs` directly in here? or do we not have the wiring yet for running e.g. anvil etc?

Reply:
That's exactly the answer: `e2e` has the scaffolding, `store-sync` doesn't. There could also be weirdness if the benchmarking tool runs a process like anvil.

The scaffold project is a great idea, I shall also give it a think